### ISSUE TITLE
CircleCI failing, bad ECMWF credentials? 

### ISSUE THREAD

**Issue by jlmaurer** [dbekaert/RAiDER#127]
Hey @dbekaert and @leiyangleon. My latest PR is passing all tests locally but failing on CircleCI when trying to download the weather model. I also see that the main dev branch is also failing at the weather model download stage with the message "Too long with no output (exceeded 10m0s): context deadline exceeded." I'm wondering if there is a problem with CircleCI credentials for ECMWF?

**Comment by leiyangleon** [comment]
I have noticed that using ECMWF api to download data takes way longer than the other apis. Also it varies. Sometimes it is pretty fast but sometime not. I checked their FAQ, and there seems to be a queue of submitted tasks. Whenever there is a long queue, it takes much longer. I don't think we actually downloaded any data using ECMWF api in CircleCI. Only including ECMWF credentials seems fine to me. Not sure what exactly happens.
